app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.order(:name)
```

Comment:
I would move the logic for getting and ordering by name to the `Ingredient` model and make it a class method. This is a bit too much logic for the controller. That said, the ActiveRecord looks good! Nice work!